docs/project_mouser/about.md

```diff
@@ -13,15 +14,16 @@ Created mainly for SLU researchers, this software is used for collecting and ana
 
 - **Source Code:** [https://github.com/oss-slu/Mouser](https://github.com/oss-slu/Mouser) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/Mouser)
 - **Client:** Dr. Karoly Toth
-- **Current Tech Lead:** Logan Wyas [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/loganwyas)
+- **Current Tech Lead:** Jack Belyeu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/jackbelyeu)
 - **Developers:**
 
-  - Ryan Carmody (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/rc10283) 
+  - Ryan Carmody (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/rc10283)
   - Andrew Obermiller (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/aobermiller)
   - Stanley Yang (capstone) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/stanleyyang2001)
   - Uday Prasad Aitha (alumni, previous tech lead) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/aithaprasad)
-  - Tyler Bush (alumni) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/tbush03) 
-  - Cori Diaz (alumni) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/coridiaz) 
+  - Tyler Bush (alumni) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/tbush03)
```

Comment:
since we don't have dates attached here, i'm thinking maybe we should alphabetize these. do you have any thoughts on what that should look like?